examples/text_labels_and_annotations/font_table.py

```diff
@@ -0,0 +1,112 @@
+"""
+==========
+Font table
+==========
+
+Matplotlib's font support is provided by the FreeType library.
+
+Here, we use `~.Axes.table` to draw a table that shows the glyphs by Unicode
+codepoint. For brevity, the table only contains the first 256 glyphs.
+
+The example is a full working script. You can download it and use it to
+investigate a font by running ::
+
+    python font_table.py /path/to/font/file
+"""
+
+import unicodedata
+
+import matplotlib.font_manager as fm
+from matplotlib.ft2font import FT2Font
+import matplotlib.pyplot as plt
+
+
+def print_glyphs(font):
+    """Print the all the glyphs in the given FT2Font font to stdout."""
+    charmap = font.get_charmap()
+    max_indices_len = len(str(max(charmap.values())))
+
+    print("The font face contains the following glyphs:")
+    for char_code, glyph_index in charmap.items():
+        char = chr(char_code)
+        name = unicodedata.name(
+                char,
+                f"{char_code:#x} ({font.get_glyph_name(glyph_index)})")
+        print(f"{glyph_index:>{max_indices_len}} {char} {name}")
+
+
+def draw_font_table(path, print_all=False):
```

Comment:
Make print_all kwonly?  (Not that it really matters here but it's the kind of arg that should likely be kwonly if this was a general-use function...)